src/main/scala/com/evolutiongaming/cluster/pubsub/OptimiseSubscribe.scala

```diff
@@ -32,15 +32,15 @@ object OptimiseSubscribe {
   type Listener[F[_]] = OnMsg[F, Any]
 
 
-  def of[F[_] : Concurrent : Parallel]: F[OptimiseSubscribe[F]] = {
+  def of[F[_] : Async: Parallel]: F[OptimiseSubscribe[F]] = {
```

Comment:
`Async` can be replaced with `Concurrent` + `Runtime`. `Async` is the most powerful typeclass in CE3, combining capabilities from both `Sync` and the rest of the hierarchy, that's why I would prefer to replace `Async` and `Sync` with the least powerful typeclasses when possible in the main code

Reply:
excellent point, addressed in [6a7aac2](https://github.com/evolution-gaming/pubsub/pull/163/commits/6a7aac2fe8fac4563d05227b068ace36a03f6978)